src/components/Typography/Editorial.js

```diff
@@ -265,16 +265,26 @@ export const link = css({
 })
 export const A = React.forwardRef(({ children, attributes, ...props }, ref) => {
   const [colorScheme] = useColorContext()
-  const colors = css({
-    color: colorScheme.text,
-    '@media (hover)': {
-      ':hover': {
-        color: colorScheme.lightText
-      }
-    }
-  })
+  const AHoverRule = useMemo(
```

Comment:
```suggestion
  const hoverRule = useMemo(
```

Variables starting with capital letters always smell of React components ;-)